test/config/database/config.ts

```diff
@@ -0,0 +1,23 @@
+function getConfig() {
+  if (process.env.CI) {
```

Comment:
use a namespace safe env var: `SENECA_KNEX_TEST_CONFIG`

Reply:
Got it